lib/icinga/externalcommandprocessor.cpp

```diff
@@ -1072,10 +1078,16 @@ void ExternalCommandProcessor::ScheduleAndPropagateTriggeredHostDowntime(double,
 void ExternalCommandProcessor::DelHostDowntime(double, const std::vector<String>& arguments)
 {
 	int id = Convert::ToLong(arguments[0]);
-	Log(LogNotice, "ExternalCommandProcessor")
-		<< "Removing downtime ID " << arguments[0];
 	String rid = Downtime::GetDowntimeIDFromLegacyID(id);
-	Downtime::RemoveDowntime(rid, true);
+
+	try {
+		Downtime::RemoveDowntime(rid, true);
+
+		Log(LogNotice, "ExternalCommandProcessor")
+				<< "Removed downtime ID " << arguments[0];
+	} catch (const invalid_downtime_removal_error& error) {
+		Log(LogWarning, error.what());
```

Comment:
This and the following calls to `Log` in this file use the error message as the facility parameter. Should be `Log(LogWarning, "ExternalCommandProcessor", error.what())`.